src/backend/distributed/utils/metadata_cache.c

```diff
@@ -2958,6 +2994,21 @@ InvalidateDistRelationCacheCallback(Datum argument, Oid relationId)
 }
 
 
+static void
+InvalidateEntireDistCache()
```

Comment:
We've a very similar function called `FlushDistTableCache()`. I can see that `InvalidateEntireDistCache ()` would be more convenient for our purpose. But, we should document our purpose with some comments